example/package.json

```diff
@@ -55,8 +55,5 @@
   },
   "jest": {
     "preset": "react-native"
-  },
-  "engines": {
```

Comment:
I would keep the engine. If this should not be here i would like to know why.

Reply:
There is a problem with keeping `yarn` version here, most likely due to some missing config in parent directory.
I don't find much benefit in restricting `yarn` on specific version in this repo. So if you think it is important, I will work on this as follow up task